sensor/common/compliance/nodescan_handler_test.go

```diff
@@ -17,14 +17,16 @@ import (
 
 // stoppable represents a gracefully stoppable thing, e.g., async process
 type stoppable struct {
-	stopC    concurrency.ErrorSignal
-	stoppedC concurrency.ErrorSignal
+	stopC     concurrency.ErrorSignal
+	stoppedC  concurrency.ErrorSignal
+	receivedC concurrency.Signal
```

Comment:
`receivedC` does not belong here as it is something very specific to what only `consumeAndCount` does.